### ISSUE TITLE
Warnings after upgrading to > 1.14.0

### ISSUE THREAD

**Issue by marioskonstantinou** [xotahal/react-native-material-ui#250]
Hello, i'm getting a few warnings after upgrading to > 1.14.0 versions. Should something else get upgraded alongside with versions >=1.15.0+ version?

<img width="248" alt="screen shot 2017-12-21 at 15 10 33" src="https://user-images.githubusercontent.com/2276860/34257215-97231a0e-e661-11e7-968b-135733981f1b.png">
<img width="245" alt="screen shot 2017-12-21 at 15 10 45" src="https://user-images.githubusercontent.com/2276860/34257216-973f31d0-e661-11e7-8a6b-1cb86fca993d.png">

**Comment by EliasJorgensen** [comment]
Bump!

I just started using this library, and i am getting similar warnings.
Would be great if these could be fixed in the next release! 😄 

@xotahal

**Comment by aburai** [comment]
@marioskonstantinou @EliasJorgensen prop types for style.leftElement and style.rightElement was from View
i changed it to PropTypes.style (see change above)

**Comment by marioskonstantinou** [comment]
@aburai Perfect! Has this fix been released or not yet?

**Comment by aburai** [comment]
@marioskonstantinou not released, but you can use my compare to edit src/ListItem/ListItem.react.js in the package under node_modules

**Comment by marioskonstantinou** [comment]
@aburai Yes i did this already and the warning is gone. But this is just a temp solution until it's released. Anw, thanks for your help.
